owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/PassCodeActivity.kt

```diff
@@ -129,7 +136,9 @@ class PassCodeActivity : BaseActivity() {
 
     private fun inflatePasscodeTxtLine() {
         val passcodeTxtLayout = findViewById<LinearLayout>(R.id.passCodeTxtLayout)
-        for (i in 0 until numberOfPassInputs) {
+        println("PACOOOOO1 " + passCodeViewModel.getPassCode()?.length)
```

Comment:
Is it necessary to print this data? Maybe it would be nice to print it through Timber? Also, if you use:
`"PACOOOOO1 ${passCodeViewModel.getPassCode ()?.length} "` is more Kotlin friendly 😄

Reply:
No, it wasn't necessary, only used with testing purposes but I forgot to remove it, thanks for reporting it and for the tip! 👍